great_expectations/render/renderer/site_builder.py

```diff
@@ -602,18 +602,20 @@ def add_resource_info_to_index_links_dict(  # noqa: PLR0913
 
         if run_id:
             filepath = (
-                os.path.join(  # noqa: PTH118
-                    "validations",
-                    *expectation_suite_name.split("."),
-                    *run_id.to_tuple(),
-                    batch_identifier,
+                "/".join(
+                    [
+                        "validations",
+                        *expectation_suite_name.split("."),
+                        *run_id.to_tuple(),
+                        batch_identifier,
+                    ]
```

Comment:
Please use `pathlib.Path` to construct the path here, rather than using raw strings.